internal/cluster/installer_test.go

```diff
@@ -42,7 +43,7 @@ var _ = Describe("installer", func() {
 		It("cluster is insufficient", func() {
 			cluster = updateClusterState(cluster, models.ClusterStatusInsufficient, db)
 			err := installerManager.Install(ctx, &cluster, db)
-			Expect(err.Error()).Should(MatchRegexp(errors.Errorf("cluster %s is expected to have exactly 3 known master to be installed, got 0", cluster.ID).Error()))
+			Expect(err.Error()).Should(MatchRegexp(regexp.QuoteMeta(errors.Errorf("cluster %s is expected to have exactly 3 known (Ready to install) master to be installed, got 0", cluster.ID).Error())))
```

Comment:
Shouldn't it be "exactly 3 (Ready to install) masters" ?

Reply:
backend still uses `Known`, so purely replacing it by the UI wording "Ready to install" would be confusing for API users